src/modules/Providers/views/details/tabs/VirtualMachines/utils/filters/concernFilter.ts

```diff
@@ -1,11 +1,12 @@
-import { EnumToTuple } from 'src/components/common/FilterGroup/helpers';
+import type { TFunction } from 'react-i18next';
+import { enumToTuple } from 'src/components/common/FilterGroup/helpers';
 
 import type { EnumValue } from '@components/common/utils/types';
 
-export const concernFilter = (t: (string) => string) => ({
+export const concernFilter = (t: TFunction) => ({
```

Comment:
I think it is easier instead of adding a type, simply import t directly here and omit it from args